### DISCUSSION TITLE
iPad videos restart when fast forwarding

### DISCUSSION THREAD

**Discussion by forthrin** [mpv-player/mpv#14197] [Q&A] [answered]
All videos recorded on iPad (Pro 12.9" 5th gen) consistently restart when pressing cursor right (fast forward). Hasn't happened for any other videos ever. Doing a stream copy with FFmpeg works around the problem. Is this a known issue?

**Comment by philipl** [comment]
https://trac.ffmpeg.org/ticket/10585

@llyyr has a fix and I'll be pushing it to FFmpeg today but it won't be in a released ffmpeg for a while. You'll need to build from source.

**Comment by forthrin** [comment]
Would test patch manually, but Firefox says server certificate is invalid. When you've pushed to FFmpeg main, leave a small note here and I'll do an mpv-build rebuild to post back if it solves the problem. Thanks!!

**Reply by kasper93** [reply]
You probably should fix your Firefox, security is important.

**Comment by forthrin** [comment]
@kasper93: Is this a Firefox issue?

> Firefox [126.0 on macOS] does not trust this site because it uses a certificate that is not valid for ffaux-bg.ffmpeg.org. The certificate is only valid for the following names: patchwork.ffmpeg.org, vote.ffmpeg.org. `[SSL_ERROR_BAD_CERT_DOMAIN]`

PS! Simply replacing the server name with one of the ones listed as valid works around the issue:

https://patchwork.ffmpeg.org/project/ffmpeg/patch/20230923111034.6320-2-llyyr.public@gmail.com/

PPS! The patch, when applied to mpv-build, seems to work. Right arrow now fast-forwards rather than restarting the iPad videos.

**Reply by kasper93** [reply]
I was thinking about the link from llyr in comments https://ffmpeg.org/pipermail/ffmpeg-devel/2023-September/314755.html, but anyway, glad you found it.

**Comment by philipl** [comment, marked as answer]
Fix has been merged to ffmpeg.
